README.md

```diff
@@ -74,16 +73,16 @@ Our [prioritized roadmap][roadmap] of features and utilities that the core team
 
 ### 0.68 - February 2023 Update
 
-In this release, we focused on releasing new features, stability and improvements.
+In this release, we have focused on releasing new features, stability, and improvements.
```

Comment:
Anything below this we can't accept,   We'll be doing a new release next week

Reply:
i am pretty sure I resolved it now.